.github/workflows/dev-smoketest.yaml

```diff
@@ -17,12 +17,18 @@ jobs:
             -v "$(pwd):/charts"
             --k3s-arg --disable=traefik@server:0
             --k3s-arg --disable=servicelb@server:0
-      - name: Run the Charts in Dev Mode
+      - name: Deploy Infra over K3D in Dev Mode
         run: |
           pushd dev && source ./.envrc
           make init
           make deploy-services
-          make deploy
```

Comment:
Does this remove deploying galoy?

Reply:
No, lnd, galoy and everything deploys in a retry thrice step after deploy-services

Reply:
https://github.com/GaloyMoney/charts/pull/1756/files#diff-84c9927dcfa9d85aa1ae62d385dc6ad49deb773c7597306f8c9dcc246d67e8baR31